nmrfx-analyst/src/main/java/org/nmrfx/analyst/peaks/Multiplets.java

```diff
@@ -184,7 +184,6 @@ public static Optional<Multiplet> mergePeaks(List<Peak> peaks) {
                 }
             }
             firstPeakDim.getMultiplet().updateCoupling(comps);
```

Comment:
I think this is a false positive by sonar, the check on line 168 ensures that there will always be atleast one iteration of the loop on line 170, which either add a new peakDim to peakDims or throw an uncaught exception, therefore there will always be at least one entry in peakDims when it does the loop on line 179 that will set firstPeakDim on the first iteration of the loop.